File: Project Euler+/Project Euler #10_ Summation of primes.c
#include <math.h>
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <assert.h>
#include <limits.h>
#include <stdbool.h>

int main()
{
    int t;
    scanf("%d", &t);
    for(int a=0;a<t;a++)
    {
        unsigned long long n,i,t1,j,sum=0;
        scanf("%d", &n);
        for(i=3;i<=n;i++)
        {
        t1=0;
        for(j=2;j<i;j++)
        {
        if(i%j==0)
        {
            t1=1;
        }
        }
        if(t1==0)
        {
            sum=sum+i;
        }
        }
        printf("%d\n", sum+2);
    }
    return 0;
}

File: Project Euler+/Project Euler #10_ Summation of primes.py
t=int(input())
for i in range(t):
    n=int(input())
    sum=0
    for i in range(3, n+1):
        t1=0
        for j in range(2, i):
            if(i%j==0):
                t1=1
                break

        if(t1==0):
            sum=sum+i
    print(sum+2)
